<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" name="Failed suite [Default suite]">
  <test thread-count="5" name="Default test(failed)">
    <classes>
<<<<<<< Updated upstream
      <class name="com.torenzo.qa.testcases.ReportsTest">
        <methods>
          <include name="launch"/>
          <include name="permission"/>
          <include name="ordercreate"/>
          <include name="login"/>
          <include name="orderpaid"/>
        </methods>
      </class> <!-- com.torenzo.qa.testcases.ReportsTest -->
      <class name="com.torenzo.qa.testcases.VerifyOrederWithNckot">
        <methods>
          <include name="launch"/>
          <include name="permission"/>
          <include name="login"/>
          <include name="verifyOrderByRecjectNckotIntoTill"/>
        </methods>
      </class> <!-- com.torenzo.qa.testcases.VerifyOrederWithNckot -->
=======
      <class name="com.torenzo.qa.testcases.CombineWithSingleGroupTest">
        <methods>
          <include name="launch"/>
          <include name="permission"/>
          <include name="login"/>
          <include name="selectTaskFromList"/>
          <include name="payment"/>
        </methods>
      </class> <!-- com.torenzo.qa.testcases.CombineWithSingleGroupTest -->
>>>>>>> Stashed changes
    </classes>
  </test> <!-- Default test(failed) -->
</suite> <!-- Failed suite [Default suite] -->
